### PR TITLE
Hotfix v2.36.1 (UPLOAD-577)

### PR DESCRIPTION
**Pull request by gniezen**
Addresses [UPLOAD-577].

[UPLOAD-577]: https://tidepool.atlassian.net/browse/UPLOAD-577

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tidepool-uploader",
   "productName": "tidepool-uploader",
-  "version": "2.35.0",
+  "version": "2.36.1",
```
Comment (cbwebdevelopment) [inline app/package.json:4]: Shouldn't a hotfix only bump the patch version, and not the minor?

**Review by cbwebdevelopment — APPROVED** [review]
LGTM, so long as the minor version upgrade was intentional.  If not, I suggest only bumping the patch version.

File: app/package.json (hunk 1 above)
Comment (gniezen) [reply app/package.json:4]: The minor version (https://github.com/tidepool-org/uploader/pull/1380) hasn't been merged yet, so this PR can supersede that one, unless you think we should merge that one first?

Comment (cbwebdevelopment) [reply app/package.json:4]: Nope, that's fine, and makes sense.

**Pull request merged**